build.sbt

```diff
@@ -118,6 +121,20 @@ lazy val zhttpTest = (project in file("zio-http-test"))
   .settings(stdSettings("zhttp-test"))
   .settings(publishSetting(true))
 
+lazy val zhttpLogging = (project in file("zio-http-logging"))
+  .settings(stdSettings("zhttp-logging"))
+  .settings(publishSetting(false))
+  .settings(
+    libraryDependencies ++= {
+      if (isScala3(scalaVersion.value)) Seq.empty
+      else Seq(reflect.value % Provided)
+    },
+  )
```

Comment:
These settings required?

Reply:
Yes, it is adding a compile time dependency on `scala-reflect` only for cala 2.x